setup.py

```diff
@@ -491,6 +491,33 @@ def check_cudnn_version_and_warn(global_option: str, required_cudnn_version: int
         )
     )
 
+    # CUDA group norm V2 is tested on SM100
+    if bare_metal_version >= Version("12.8"):
+        arch_flags = ["-gencode=arch=compute_100,code=sm_100"]
+    else:
+        arch_flags = ["-gencode=arch=compute_90,code=compute_90"]
```

Comment:
QQ: Looking at this conditions, it is a Blackwell exclusive. Is it supposed to work on Hopper?

Reply:
It works on Hopper if we add it to `arch_flags` and `DISPATCH_CUDA_ARCH_AND_LOWER_BOUND_SM_COUNT`, but we need a smart way to avoid exhausting `SM_COUNT`. This needs much code change and will not appear in this pull request.

Reply:
It works on Hopper if we add it to `arch_flags` and `DISPATCH_CUDA_ARCH_AND_LOWER_BOUND_SM_COUNT`. However, I think we need a smarter way to determine template args to support more GPUs, instead of exhausting `SM_COUNT`.